compiler/hash-typecheck/src/traverse/mod.rs

```diff
@@ -1020,10 +1109,16 @@ impl<'gs, 'ls, 'cd, 'src> visitor::AstVisitor for TcVisitor<'gs, 'ls, 'cd, 'src>
 
     fn visit_loop_block(
         &mut self,
-        _ctx: &Self::Ctx,
-        _node: hash_ast::ast::AstNodeRef<hash_ast::ast::LoopBlock>,
+        ctx: &Self::Ctx,
+        node: hash_ast::ast::AstNodeRef<hash_ast::ast::LoopBlock>,
     ) -> Result<Self::LoopBlockRet, Self::Error> {
-        todo!()
+        let walk::LoopBlock(term) = walk::walk_loop_block(self, ctx, node)?;
+
+        // Add the location of the type as the whole block
+        let location = self.source_location(node.span());
+        self.location_store_mut().add_location_to_target(term, location);
+
+        Ok(term)
```

Comment:
This should return void no? Unless there is `break x` in the loop.

Reply:
I don't think break expressions can return anything as of yet...

Reply:
Okay so then void.